chat/conversation_handlers.py

```diff
@@ -193,10 +193,13 @@ def conversation_before_save_handler(record, original_record, conn):
         if original_record is None:
             check_if_context_has_master_key(
                     "Call chat:create_conversation instead")
-        if 'admin_ids' in record:
-            del record['admin_ids']
-        if 'participant_ids' in record:
-            del record['participant_ids']
+        allowed_keys = ['title', 'metadata', 'distinct_by_participants']
+        keys_to_be_removed = [key
+                              for key in record
+                              if not key.startswith('_') and
+                              key not in allowed_keys]
+        for key in keys_to_be_removed:
+            del record[key]
```

Comment:
I think blacklist will be better than whitelist? So the user can still add attributes to a conversation for his application logic?